src/test/java/elizCurtnazar/tests/tema6/HerokuForms.java

```diff
@@ -0,0 +1,138 @@
+
+import java.io.File;
+import java.io.IOException;
+import java.util.List;
+
+public class HerokuForms {
+
+    static ChromeDriver driver = null;
+
+    public static void main(String[] args) throws IOException {
+        navigateToHerokuFormsPage();
+        inputUsername();
+        inputPassword();
+        inputTextAreaComment();
+        chooseFile();
+        checkboxItems();
+//        radioItems();
+        try {
+            radioItems();
+            System.out.println("Am intrat pe blocul de try.");
+        } catch (IndexOutOfBoundsException e) {
+            System.out.println("Am intrat de blocul de catch");
+            List <WebElement> radioButtons = driver.findElements(By.name("radioval"));
+            radioButtons.get(2).click();
+            System.out.println("Am ales radio 3.");
+            //ALTA VARIANTA DE REZOLVARE
+//            File file = driver.getScreenshotAs(OutputType.FILE);
+//            File destinationFile = new File("C:\\Users\\Deniz\\IdeaProjects\\poza2.png");
+//            FileUtils.copyFile(file, destinationFile);
+        } finally {
+            File file = driver.getScreenshotAs(OutputType.FILE);
+            File destinationFile = new File("C:\\Users\\Deniz\\IdeaProjects\\poza2.png");
+            FileUtils.copyFile(file, destinationFile);
+            System.out.println("Am facut screenshot de pe blocul de finally.");
+            //ALTA VARIANTA DE REZOLVARE
+//            driver.quit();
+//            System.out.println("Am inchis browserul de pe blocul de finally.");
+        }
+        selectValues();
+        dropdown();
+        scrollSubmitButton();
+        closeBrowser();
+    }
+
+    public static void navigateToHerokuFormsPage(){
+        driver = BrowserManager.createChromeDriver();
+        driver.get("https://testpages.herokuapp.com/styled/basic-html-form-test.html");
+        System.out.println("Am deschis pagina Heroku forms page");
+    }
+
+    public static void inputUsername() {
+        WebElement username = driver.findElement(By.xpath("//*[@id=\"HTMLFormElements\"]/table/tbody/tr[1]/td/input"));
+        username.click();
+        username.clear();
+        username.sendKeys("eliz.Curtnazar");
+    }
+
+    public static void inputPassword() {
+        WebElement password = driver.findElement(By.name("password"));
+        password.click();
+        password.clear();
+        password.sendKeys("curtnazar91");
+        System.out.println("Am adaugat password ul.");
+    }
+
+    public static void inputTextAreaComment() {
+        WebElement inputComment = driver.findElement(By.name("comments"));
+        inputComment.click();
+        inputComment.clear();
+        inputComment.sendKeys("Aceasta este tema 6.");
+        System.out.println("Am adaugat un comment.");
+    }
+
+    public static void chooseFile() {
+        WebElement file = driver.findElement(By.name("filename"));
+        file.sendKeys("C:\\Users\\Deniz\\Desktop\\eliz\\git.txt");
```

Comment:
Try to avoid using local paths when adding files. This won't work on other machines. Create the file inside the repo, in your package for homework 6 and get the path from there. Thanks!

Reply:
done.